packages/@aws-cdk/aws-lambda/lib/lambda-insights.ts

```diff
@@ -31,6 +32,11 @@ export abstract class LambdaInsightsVersion {
    */
   public static readonly VERSION_1_0_98_0 = LambdaInsightsVersion.fromInsightsVersion('1.0.98.0');
 
+  /**
+   * Version 1.0.119.0
+   */
+  public static readonly VERSION_1_0_119_0 = LambdaInsightsVersion.fromInsightsVersion('1.0.119.0');
```

Comment:
Still need to confirm that the versions are the same for ARM & x86. The version is currently only listed on the x86 docs.

Reply:
Confirmed.